### ISSUE TITLE
Python program stuck after debugging finish (cursor reach end of program)

### ISSUE THREAD

**Issue by AlternativeSolution** [vim-vdebug/vdebug#505]
Python program stuck after cursor reaches end of program. Vim with vdebug plugin work correct.
I run program use command:  `python3 -S /usr/local/lib/Komodo-PythonRemoteDebugging-12.0.1-91869-linux-x86_64/pydbgp -d localhost:9000 main.py` 
However, if I use `python debugger.close()`(\<F6\>) or `python debugger.detach()`(\<F7\>) python program is finishing.
I want to python program finish after debugger cursor reach end of program.
What I have to do?
